tests/testthat/test-meta_validate.R

```diff
@@ -0,0 +1,48 @@
+meta <- meta_dummy()
```

Comment:
The test item in the test plan is not tested. i.e., "use meta_dummy() as input. Will return the same meta" in https://github.com/Merck/metalite/issues/32

Reply:
so like: `expect_equal(meta_validate(meta_dummy()), meta)` ?